otaserver/static/otaserver.html

```diff
@@ -47,9 +47,10 @@
     bodyFormData.set('publish_id', id);
     bodyFormData.set('urls', deviceAddr);
     bodyFormData.set('version', version);
+    var notify_path = '{{ root_url }}' === '' ? 'notify' : '{{ root_url }}' + '/notify' ;
     axios({
         method: 'post',
-        url: 'http://{{ host }}:{{ port }}/notifyv4',
+        url: 'http://{{ host }}:{{ port }}/' + notify_path,
```

Comment:
```suggestion
        url: 'http://{{ host }}:{{ port }}/{{ notify_path }}',
```

(untested but should work and it's cleaner)